cocotb/triggers.py

```diff
@@ -420,6 +420,10 @@ def clear(self):
         :meth:`~cocotb.triggers.Event.set` is called again."""
         self.fired = False
 
+    def is_set(self):
```

Comment:
```suggestion
    def is_set(self) -> bool:
```
while you're here, maybe